Feature/#13 2.x port

oechsler opened the pull request:
Port of #21 
Closes #13

Commit:
Add givenEntitiyIsCreated method to test trait

Commit:
Update entity tests

Entity tests now make use of the givenEntityIsCreated method

Commit:
FIx typo in i18n/de.yml

Commit:
Update de.yml to suggested translation

The is_created key now translates to "WurdeErzeugt" in de.yaml

msvujnovic-becklyn commented on src/Resources/skeleton/ddd/entity-test-trait.tpl.php:
```diff
@@ -51,4 +51,14 @@ protected function <?= $i18n["test"]["_given"]; ?><?= $entity; ?><?= $i18n["test
     }
+
+    protected function <?= $i18n["test"]["_given"]; ?><?= $entity; ?><?= $i18n["test"]["is_created"]; ?> (
+        ?<?= $entity; ?>Id $<?= $strtocamel($entity); ?>Id = null
```
Please put a comma at the end of this line, because PHP8 supports it if it ends up being the only argument

Commit:
Add comma to parameter list

Intentional usage of the PHP 8 syntax here.
In almost every case multiple parameters have to be added to this list.
